ewm-main-service/src/main/java/ru/practicum/main/controller/publicapi/PublicCategoryController.java

```diff
@@ -0,0 +1,35 @@
+package ru.practicum.main.controller.publicapi;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+import ru.practicum.main.dto.category.CategoryDto;
+import ru.practicum.main.service.category.CategoryService;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/categories")
+@RequiredArgsConstructor
+@Slf4j
+@Validated
+public class PublicCategoryController {
+
+    private final CategoryService categoryService;
+
+    @GetMapping
+    public List<CategoryDto> getCategories(
+            @RequestParam(defaultValue = "0") int from,
+            @RequestParam(defaultValue = "10") int size) {
```

Comment:
Можно везде в контроллерах использовать классы-обертки,
сейчас большой разницы не будет, но если захотим использовать RequestParam(reqiured = false), то не сможем принимать null значения, потому что примитивы (int, long, boolean и др.) не могут быть null

```suggestion
            @RequestParam(defaultValue = "10") Integer size) {
```